website/src/documentation/runners/flink.md

```diff
@@ -286,6 +287,9 @@ To run on a separate [Flink cluster](https://ci.apache.org/projects/flink/flink-
 </span>
 
 <span class="language-py">3. Submit the pipeline as above.
+Note however that when you want to run your pipeline on a distributed/remote Flink cluster, you will
+have to change the `environment_type` option.
+See [here]({{ site.baseurl }}/roadmap/portability/#sdk-harness-config) for details.
```

Comment:
Should mention here that the default environment_type is `DOCKER` and there is no need to specify anything for `environment_type` if that deployment option is ok.

Reply:
I changed the instructions to use loopback, so docker would be a change from that.

Reply:
I saw that. I just think that it would be fair to point out the default is Docker and the LOOPBACK is just for experimentation.

Reply:
Okay, I made that more explicit.